crates/holochain/src/conductor/conductor.rs

```diff
@@ -816,10 +817,13 @@ mod network_impls {
                     let sender = self.p2p_batch_sender(&dna_hash);
                     let (result_sender, response) = tokio::sync::oneshot::channel();
                     let _ = sender
-                        .send(P2pBatch {
-                            peer_data,
-                            result_sender,
-                        })
+                        .send_timeout(
```

Comment:
Not okay to risk blocking here, this is called by the p2p event dispatch loop and risks killing Holochain